fighthealthinsurance/payer_policy_parsers.py

```
@@ -0,0 +1,75 @@
+
+
+@dataclass(frozen=True)
+class ParsedPolicyEntry:
+    title: str
+    url: str
+    payer_policy_id: str = ""
+
+
+# Cigna A-Z titles look like "Acupuncture - (CPG024)" -- pull the id off the
+# end so we can use it for cross-references.
+_CIGNA_ID_SUFFIX_RE = re.compile(r"\s*[-–]\s*\(([A-Za-z0-9_]+)\)\s*$")
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)